### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by georgewrmarshall**
# Release 1.0.0

## Description
This PR prepares the release of version 1.0.0 of the design system packages.

## Changes

- feat: color updates to the design tokens package ([#230](https://github.com/metamask/metamask-design-system/pull/230))

- `@metamask/design-tokens` package migrated from standalone repository into the design system monorepo ([128](https://github.com/MetaMask/metamask-design-system/pull/128))

## Release Process Checklist
- [x] Branch named `release/1.0.0` created
- [x] Changelogs updated and validated
- [x] Dependencies updated to latest versions (`yarn constraints --fix && yarn && yarn dedupe`)
- [x] PR review and testing completed
- [x] Ready for "Squash & Merge" with commit message "Release 1.0.0"

## Post-Merge Steps
1. The `publish-release` GitHub Action will tag the release
2. NPM publishing requires approval from the `npm-publishers` team
3. Verify package publication on [NPM](https://npms.io/search?q=scope%3Ametamask)

### CODE REVIEW

File: packages/design-tokens/CHANGELOG.md [hunk 1]
```diff
@@ -310,44 +314,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Initial release.
-
-- Initial release.
```
Comment (georgewrmarshall) [inline packages/design-tokens/CHANGELOG.md:315]: Removing the duplicate Initial release

File: packages/design-tokens/CHANGELOG.md [hunk 2]
```diff
@@ -310,11 +314,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Initial release
```
Comment (georgewrmarshall) [inline packages/design-tokens/CHANGELOG.md:317]: Removing the duplicate Initial release

File: packages/design-tokens/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/design-tokens",
-  "version": "4.1.0",
+  "version": "4.2.0",
```
Comment (georgewrmarshall) [inline packages/design-tokens/package.json:3]: Bumping design tokens minor version to include new background-muted tokens

File: packages/design-tokens/CHANGELOG.md [hunk 4]
```diff
@@ -7,9 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ## [Unreleased]
 
+## [4.2.0]
+
 ### Added
 
-- Migrate `@metamask/design-tokens` into the design system monorepo ([128](https://github.com/MetaMask/metamask-design-system/pull/128))
+- feat: color updates to the design tokens package ([#230](https://github.com/metamask/metamask-design-system/pull/230))
+
+- `@metamask/design-tokens` package migrated from standalone repository into the design system monorepo ([128](https://github.com/MetaMask/metamask-design-system/pull/128))
```
Comment (georgewrmarshall) [inline packages/design-tokens/CHANGELOG.md:16]: Just re-wording second entry to migrate design tokens package

**Review by brianacnguyen — APPROVED** [review]

**Pull request merged**